sdk/src/main/scala/com/horizen/account/certificatesubmitter/AccountCertificateSubmitter.scala

```diff
@@ -39,29 +46,41 @@ class AccountCertificateSubmitter(settings: SidechainSettings,
   type MP = AccountMemoryPool
   type PM = AccountBlock
 
-  override def getUtxoMerkleTreeRoot(referencedEpoch: Int, state: AccountState): Optional[Array[Byte]] = Optional.empty()
-
   override def getWithdrawalRequests(state: AccountState, referencedEpochNumber: Int): Seq[BackwardTransfer] =
```

Comment:
no one is using this method, it was replaced by `AbstractState.backwardTransfers`.
So we may remove it from here, from base class and from UTXO class as it was done for `getUtxoMerkleTreeRoot`